core/operation/src/main/java/uk/gov/gchq/gaffer/operation/Operation.java

```diff
@@ -28,7 +30,12 @@
  * Operations must be JSON serialisable in order to make REST API calls.
  */
 @JsonTypeInfo(use = JsonTypeInfo.Id.CLASS, include = JsonTypeInfo.As.PROPERTY, property = "class")
-public interface Operation {
+public interface Operation extends Closeable {
+
+    default void close() throws IOException {
```

Comment:
Are all standard gaffer operations going to implement this method?  Or is this something that's going to be done by exception rather than default?